grpc/swift-cubic/Sources/swift-cubic/CubicManager.swift

```diff
@@ -0,0 +1,168 @@
+//
+//  CubicManager.swift
+//  cubic-demo
+//
+//
+
+import Foundation
+import SwiftProtobuf
+import GRPC
+import NIO
+import NIOSSL
+
+public typealias CubicFailureCallback = (_ error: Error) -> ()
+
+public class CubicManager {
```

Comment:
Other languages call this higher level unit "Client", perhaps we could rename this to Client as well?

Reply:
The basic client class generated by GRPC is called Cobaltspeech_Cubic_CubicServiceClient. This is its extension for simple usage in Swift. I wouldn't name it Client, because it can conflict with other classes in customer's source code. I agree that CubicManager is not very suitable name, because it is not a manager.

Reply:
CubicClient is fine; but I wonder why there would be a conflict? Does swift not have namespaces based on imported packages?

e.g. `cubic.Client` in Python